code/modules/gear_presets/survivors/lv_759/preset_hybrisa.dm

```diff
@@ -203,18 +204,23 @@
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(new_human), WEAR_FEET)
 	..()
 
+
+//////////////// WEYLAND YUTANI /////////////////////
+////////////////////////////////////////////////////
+
 // Weyland Yutani Corpo
 
 /datum/equipment_preset/survivor/corporate/hybrisa
 	name = "Survivor - Hybrisa - Corporate Liaison"
 	assignment = "Corporate Liaison"
 
 /datum/equipment_preset/survivor/corporate/hybrisa/load_gear(mob/living/carbon/human/new_human)
+
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/liaison_suit/ivy(new_human), WEAR_BODY)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/vest(new_human), WEAR_JACKET)
 	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/satchel/lockable(new_human), WEAR_BACK)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/laceup(new_human), WEAR_FEET)
-	..()
```

Comment:
removing this will mean they don't get a weapon, metal, headset, etc. I doubt you want to remove the callback to parent.

Reply:
Ah yeah not sure why I did. It should be fixed now.